test/Api.Tests/ControllerTests/PermissionsControllerTest.cs

```diff
@@ -105,7 +89,7 @@ public void IsAllowedToCreateTestForbidden()
             _tester.TestController(_tester.Student.Email)
                 .Calling(c => c.IsAllowedToCreate())
                 .ShouldReturn()
-                .StatusCode(403);
+                .StatusCode(403); // Not able to use Forbidden() as the original method returns the status code only but not Forbid()
```

Comment:
in this case, this is an error in the `IsAllowedToCreate` method and should be changed there